understanding/20/labels-or-instructions.html

```diff
@@ -24,47 +23,44 @@ <h2>Intent of Labels or Instructions</h2>
       
       <p>The intent of this Success Criterion is not to clutter the page with unnecessary information
          but to provide important cues and instructions that will benefit people with disabilities.
-         Too much information or instruction can be just as much of a hindrance as too little.
+         Too much information or instruction can be just as harmful as too little.
          The goal is to make certain that enough information is provided for the user to accomplish
          the task without undue confusion or navigation.
       </p>
-      
-      <div class="note">
-         
-         <p>When labels are provided for input objects, the input object's relationship to the
-            label (or to redundant text serving as the label) must be programmatically determinable
-            or available in text per 
-            <a href="info-and-relationships">1.3.1: Info and Relationships</a>.
-         </p>
          
-      </div>
-      
-      
+      <p>This Success Criterion does not require that labels or instructions be correctly marked up or
+         identified/associated with their respective controls - this aspect is covered separately by
+         <a href="info-and-relationships">1.3.1: Info and Relationships</a>. It is possible for content
+         to pass this Success Criterion (providing relevant labels and instructions) while failing
+         Success Criterion 1.3.1 (if the labels or instructions aren't correctly marked up/identified/associated).
```

Comment:
I would like to avoid the "this/this/this" construction

Reply:
maybe just separate with commas

Reply:
not quite clear what you mean. the use of "this" in that one paragraph? or the fact there are three paragraphs starting with "This...", "Further, this..." and "While this..."?

Reply:
"(if the labels or instructions aren't correctly marked up, identified, or associated)"

Reply:
ah, gotcha. fixed and force-pushed an update